applications/asset_tracker/CMakeLists.txt

```diff
@@ -31,6 +32,9 @@ add_subdirectory(src/ui)
 add_subdirectory(src/cloud_codec)
 add_subdirectory(src/gps_controller)
 add_subdirectory(src/env_sensors)
+if (CONFIG_BH1749)
```

Comment:
Typically such conditions are handled with `add_subdirectory_ifdef(CONFIG_BH1749 src/light_sensor)`. Technically the same, but in a more concise form.

Reply:
yes that is better, updated.